src/test/java/uk/gov/hmcts/probate/transformer/CallbackResponseTransformerTest.java

```diff
@@ -771,26 +770,26 @@ public void shouldTestForNullDOD() {
         assertEquals(null, callbackResponse.getData().getDeceasedDateOfDeath());
     }
 
-    @Test
-    public void shouldConvertRequestToDataBeanForPaymentWithCheque() {
-        CaseData caseData = caseDataBuilder.solsPaymentMethods(SOL_PAY_METHODS_CHEQUE)
-                .build();
-        when(caseDetailsMock.getData()).thenReturn(caseData);
-
-        when(feeServiceResponseMock.getFeeForNonUkCopies()).thenReturn(feeForNonUkCopies);
-        when(feeServiceResponseMock.getFeeForUkCopies()).thenReturn(feeForUkCopies);
-        when(feeServiceResponseMock.getApplicationFee()).thenReturn(applicationFee);
-        when(feeServiceResponseMock.getTotal()).thenReturn(totalFee);
-
-        CallbackResponse callbackResponse = underTest.transformForSolicitorComplete(callbackRequestMock, feeServiceResponseMock);
-
-        assertCommon(callbackResponse);
-        assertLegacyInfo(callbackResponse);
-        assertEquals(NO, callbackResponse.getData().getBoEmailRequestInfoNotification());
-        assertEquals(TOTAL_FEE, callbackResponse.getData().getTotalFee());
-        assertEquals(SOL_PAY_METHODS_CHEQUE, callbackResponse.getData().getSolsPaymentMethods());
-        assertNull(callbackResponse.getData().getSolsFeeAccountNumber());
-    }
+//    @Test
```

Comment:
should this be removed?

Reply:
removing the test which tests for sols cheque payment method which im removing now